examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,74 @@
+"""
+Extracting brain signal from spheres
+====================================
+
+This example estimates a connectivity between Default Mode Network components
+using spheres as ROIs.
+
+"""
+
+# Fetching datasets ###########################################################
+from nilearn import datasets
+adhd_dataset = datasets.fetch_adhd(n_subjects=1)
+
+# print basic information on the dataset
+print('First subject functional nifti image (4D) is at: %s' %
+      adhd_dataset.func[0])  # 4D data
+
+
+# Extracting region signals ###################################################
+from nilearn import image
+from nilearn import input_data
+
+from sklearn.externals.joblib import Memory
+mem = Memory('nilearn_cache')
+
+# Coordinates of Default Mode Network
+dmn_coords = [(0, -52, 18), (-46, -68, 32), (46, -68, 32), (0, 50, -5)]
+labels = [
+    'Posterior Cingulate Cortex',
+    'Left Temporoparietal junction',
+    'Right Temporoparietal junction',
+    'Medial prefrontal cortex'
+]
+
+masker = input_data.NiftiSpheresMasker(
+    dmn_coords, radius=8,
+    detrend=True, standardize=True,
+    low_pass=None, high_pass=0.01, t_r=2.5,
+    memory=mem, memory_level=1, verbose=2)
```

Comment:
Looks much better. I also replaced precision matrix by covariance since it makes more sense in that example.